### PR TITLE
Check for critical imports in swift files

### PR DESCRIPTION
**Pull request by rasmustautsglia**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-2450

**What was solved?**
This PR takes an alternative approach to Swiftlint usage and uses script instead to determine if critical imports are present.

If SwiftUI is imported, UIKit nor Foundation is needed. If not, then either of them has to be in a swift file. When there isn't one, Xcode will display a warning.

**Release notes:**

 - [ ] Feature
 - [X] Ignore
 - [ ] Release notes (Is it clear from the description here?)
 - [ ] Migration guide (If changes are needed for integrator already using the SDK - what needs to be communicated? Add underneath please)

**Additional info:**

 - [ ] Tests fixed, added? Unit, acceptance, snapshots?
 - [ ] Logging necessary for future troubleshooting of customer issues added?

**Screenshots:**

### CODE REVIEW

**Review by gersonnoboa — COMMENTED** [review]
Does this run only on Xcode? One of the benefits of doing it on SwiftLint was that if there's something wrong, then it would show up on GitHub as a comment. This would only show up when you are developing. However yes, SwiftLint sucks so I don't know exactly what to do.

**Comment by rasmustautsglia** [comment]
Swiftlint is great for checking elements that are present. However, it has its downside, which is that it cannot detect what is absent. I didn't find a possibility to write a rule that checks for missing imports.

**Review by gersonnoboa — APPROVED** [review]
Well at least now the warning will be there :)

**Review by igorkravchenko — COMMENTED** [review]
Does this change imply that UIKit has to be always imported?

**Comment by rasmustautsglia** [comment]
This change implies that either UIKit or Foundation has to be always imported for .swift files. Currently, there is about 110 files that don't have either

File: bitrise.yml [hunk 1]
```diff
@@ -173,6 +173,10 @@ workflows:
             make clone-snapshots-ci repo_url="https://$GITHUB_USERNAME:$GITHUB_API_TOKEN@$SNAPSHOTS_REPO"
     - cache-pull@2: {}
     - certificate-and-profile-installer@1: {}
+    - script@1:
+        inputs:
+        - content: |-
+            bundle install
```
Comment (gersonnoboa) [inline bitrise.yml:179]: Perhaps this change means that all workflows that build the app need to have a `bundle install` now, not only the one that runs =checks in the PRs.

File: Podfile [hunk 2]
```diff
@@ -1,4 +1,6 @@
 source 'https://github.com/CocoaPods/Specs.git'
+plugin 'cocoapods-no-autoimports'
+# cocoapods-no-autoimports__pods = ['Pods-GliaWidgets.debug']
```
Comment (gersonnoboa) [inline Podfile:3]: Is this needed for something?

File: Gemfile [hunk 3]
```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gem "fastlane"
+gem 'cocoapods-no-autoimports'
```
Comment (gersonnoboa) [inline Gemfile:4]: GitHub is complaining about a lack of a new line here.

File: Podfile (hunk 2 above)
Comment (rasmustautsglia) [reply Podfile:3]: Yes it is a critical part of the plugin

**Review by EgorovEI — APPROVED** [review]

File: Gemfile (hunk 3 above)
Comment (rasmustautsglia) [reply Gemfile]: fixed [a00e1de](https://github.com/salemove/ios-sdk-widgets/pull/838/commits/a00e1de203c793d83b3c7bb06c8f106ef53bf89d)

File: bitrise.yml (hunk 1 above)
Comment (rasmustautsglia) [reply bitrise.yml:179]: This commit will be dropped. Thank you for the notice

**Review by igorkravchenko — APPROVED** [review]

**Pull request merged**